salt/roster/cache.py

```diff
@@ -28,7 +28,6 @@
 overriding the one in ``roster_defaults``:
 
 .. code-block:: yaml
-
 roster_order:
```

Comment:
I think for these to display as code blocks, the block of yaml lines needs to be indented beneath the code-block line? Like this:
```
.. code-block:: yaml
    roster_order:
      host: id
```